lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -197,11 +201,23 @@ final class AddComicViewController: UIViewController, AlertDisplaying {
         }
     }
 
+    private func getMirroredDropDownImage(button: UIButton, picker: UIPickerView) -> UIImage {
+        guard
```

Comment:
This is very clever but you can probably just set the button's `isSelected` property and then set in IB different images for selected and not selected. The only reason I'd suggest one over the other is because this would indicate to me something more than just "the arrow needs to be flipped when the picker is down". This is very clear, well-written code, so good job!